lasagne/nonlinearities.py

```diff
@@ -269,6 +269,70 @@ def elu(x):
     return theano.tensor.switch(x > 0, x, theano.tensor.expm1(x))
 
 
+
+# selu
+class SELU(object):
+    """Scaled Exponential Linear Unit
+    :math:`\\varphi(x) = \\lambda (x > 0 ? x : \\alpha(e^x-1)`
+
+    The Scaled Exponential Linear Unit (SELU) was introduced in [1]_
+    as an activation function that allows the construction of
+    self-normalizing neural networks.
+
+    Parameters
+    ----------
+    scale : float32
+        The scale parameter :math:`\\lambda` for scaling all output.
+
+    scale_neg  : float32
+        The scale parameter :math:`\\alpha`
+        for scaling output for negative argument values.
+
+    Methods
+    -------
+    __call__(x)
+        Apply the SELU function to the activation `x`.
+
+    Examples
+    --------
+    In contrast to other activation functions in this module, this is
+    a class that needs to be instantiated to obtain a callable:
+
+    >>> from lasagne.layers import InputLayer, DenseLayer
+    >>> l_in = InputLayer((None, 100))
+    >>> from lasagne.nonlinearities import SELU
+    >>> selu = SELU(2, 3)
+    >>> l1 = DenseLayer(l_in, num_units=200, nonlinearity=selu)
+
+    See Also
+    --------
+    selu: Instance with :math:`\\alpha=1.6733, \\lambda=1.0507`, as recommended in [1]_.
```

Comment:
Let's stay more neutral and say `as used in [1]_.` They don't necessarily recommend it, at least not so generally.